### PR TITLE
Improve the default config to add additional discovery options

### PR DESCRIPTION
**Pull request by cangencer**

### CODE REVIEW

File: hazelcast-jet-distribution/src/root/config/hazelcast.yaml [hunk 1]
```diff
@@ -1,30 +1,78 @@
...
+        label: application=hazelcast-jet
+        zones: us-east1-b,us-east1-c
+    # Allows access to cluster via the REST API
+    rest-api:
+      enabled: true
+      endpoint-groups:
+        CLUSTER_READ:
+          enabled: true
+        CLUSTER_WRITE:
+          enabled: true
```
Comment (gurbuzali) [inline hazelcast-jet-distribution/src/root/config/hazelcast.yaml]: this enables cluster shutdown and it is `disabled` by default for IMDG, is it a good idea to enable it by default ?

Comment (cangencer) [reply hazelcast-jet-distribution/src/root/config/hazelcast.yaml]: It is useful for doing shutdown from MC and using the cluster.sh scripts. I thought we enable it by default so those scripts work.

**Comment by cangencer** [comment]
verify

**Comment by cangencer** [comment]
verify

**Pull request merged**